addons/http_routing/models/ir_http.py

```diff
@@ -547,10 +547,10 @@ def _pre_dispatch(cls, rule, args):
         if request.is_frontend:
             cls._frontend_pre_dispatch()
 
-        # update the context of "<model(...):...>" args
-        for key, val in list(args.items()):
-            if isinstance(val, models.BaseModel):
-                args[key] = val.with_context(request.context)
+            # update the context of "<model(...):...>" args
+            for key, val in list(args.items()):
+                if isinstance(val, models.BaseModel):
+                    args[key] = val.with_context(request.context)
```

Comment:
?? The commit message is really lacking some explanation.. 

> when not request.is_frontend
the env of args don't need to be synchronized with the request again

The most important thing is.. why? Always write one in your commit message
Right now, you will be the only one to figure why was this change needed.

Reply:
Becaue it has been synced in the `super()`.
When you override the method, if there is no customization (`cls._frontend_pre_dispatch()`) you don't need to do it **again**

Reply:
Nice, that would really need to be part of the commit message :+1: 
No need to change here since it was r+ but remind it for future commits

Reply:
I will add these info the description of the PR